lib/querystring.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+const {
+  StorageObject,
+  percentEscape,
+  unescapeBuffer
+} = require('internal/querystring');
 const QueryString = module.exports = {
   unescapeBuffer,
```

Comment:
Hmm..mind adding a comment about that they have never been documented, so technically not public APIs, just exposed by accident?